### ISSUE TITLE
Skip to main content link bypasses important content

### ISSUE THREAD

**Issue by danchamp** [localgovdrupal/localgov_theme#149]
The "Skip to main content" link currently targets an empty `<a>` element which is the first child of the `<main>` element:  [page.html.twig](https://github.com/localgovdrupal/localgov_theme/blob/fdcdc6701855aafe0c4b62f618b8995b7a331971/templates/system/page.html.twig#L123)

Taking this service page as an example: https://localgovdrupal.agile.coop/rubbish-and-recycling

Using the skip link moves the focus below the main page heading, introduction, and task links. This is important content that should be considered part of the main page content.

One option is to give the first `<div>` below the breadcrumbs the id 'main-content', and remove the existing target `<a>` element, i.e:

`<div id="main-content" tabindex="-1" {{ pageAttributes.addClass('row') }}>`

This will include messages, content-top, and sidebar-first regions in the main page content - the sidebar may be an issue and require its own skip link.

**Comment by paulpopus** [comment]
Another issue that I see with the current structure of it is that our main has a section with no H1 usually. So the page template looks something like this
```
h1-{pagetitle}
main-{skip-to-content}
  section
    {content}
```
Which I don't think is strictly wrong, maybe just confusing but coupled with the skip to content issue of skipping the page title altogether we can probably move the main up to encompass the page title and the other blocks (such as admin editorial blocks), but this feels like the wrong approach because we're just adding in all blocks and allows misuse.

The content top is in a div, should this be in a `<section>` instead?

My immediate remedy for this is to just move the Page title block into the content block area, is there a concern with this? @danchamp 

It would look a bit like this
```
main-{skip-to-content}
  section
    h1-{pagetitle}
    {content}
```

**Comment by markconroy** [comment]
You can see what I have done to solve this here:
https://github.com/localgovdrupal/localgov_base/issues/23
